extras/create_send_notifications.py

```diff
@@ -106,13 +121,13 @@ def main():
     # Change to the correct output directory
     os.chdir(os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR))
 
-    # Build list of CyHy orgs
-    cyhy_org_ids = build_cyhy_org_list(db)
-    logging.debug("Found {} CYHY orgs: {}".format(len(cyhy_org_ids), cyhy_org_ids))
+    # Build list of orgs that should receive notifications
+    notification_org_ids, notifications_to_delete = build_notifications_org_list(db)
+    logging.debug("Will attempt to generate notifications for {} orgs: {}".format(len(notification_org_ids), notification_org_ids))
 
     # Create notification PDFs for CyHy orgs
     master_report_key = Config(args["CYHY_DB_SECTION"]).report_key
-    num_pdfs_created = generate_notification_pdfs(db, cyhy_org_ids, master_report_key)
+    num_pdfs_created = generate_notification_pdfs(db, notification_org_ids, master_report_key)
```

Comment:
Consistent with variable change suggestion.

```suggestion
    num_pdfs_created = generate_notification_pdfs(db, notifications_org_ids, master_report_key)
```